go/chat/utils/utils.go

```diff
@@ -703,25 +700,15 @@ func PickLatestMessageUnboxed(conv chat1.ConversationLocal, typs []chat1.Message
 }
 
 func GetConvMtimeLocal(conv chat1.ConversationLocal) gregor1.Time {
-	timeTyps := []chat1.MessageType{
-		chat1.MessageType_TEXT,
-		chat1.MessageType_ATTACHMENT,
-		chat1.MessageType_SYSTEM,
-	}
-	msg, err := PickLatestMessageUnboxed(conv, timeTyps)
+	msg, err := PickLatestMessageUnboxed(conv, VisibleChatMessageTypes())
```

Comment:
I'd like to keep these lists here, I'm a little worried about the scope of stuff that gets changed by modifying `VisibleChatMessageTypes`

Reply:
I think the chances of a bug coming about by keeping the lists separate is greater than the chance when they're unified. So I'd argue this is a good idea.

If we keep the list unified then anyone changing it needs to remember to grep for it and if they're change is nuanced may need to split a new list.

If we keep them separate there's no chance of someone remembering where all the copies of the list are to update them.